app/src/main/java/org/dhis2/utils/customviews/CustomTextView.java

```diff
@@ -223,7 +223,7 @@ public void setEditable(Boolean editable) {
         editText.setFocusable(editable);
         editText.setFocusableInTouchMode(editable);
         editText.setClickable(editable);
-        editText.setEnabled(editable);
+//        editText.setEnabled(editable);
```

Comment:
Is this needed??

Reply:
Nop, forgot to put it back... chaning it now